### PR TITLE
Create strings.xml for Chinese

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

**Review by FunkyMuse — CHANGES_REQUESTED** [review]
Thanks for the PR

Just add one more line

<string name="chinese">translation</string>

File: app/src/main/res/values-cn/strings.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<resources>
+    <string name="app_name" translatable="false">Vigilante</string>
```
Comment (FunkyMuse) [inline app/src/main/res/values-cn/strings.xml:2]: you won't be needing the app name, translatable is false, you can delete it

Comment (sr093906) [reply app/src/main/res/values-cn/strings.xml:2]: @FunkyMuse I added <string name="chinese">translation</string>. Could I keep the line of the app name? I'd like to keep the line number of strings,xml for Chinese same as raw strings.xml for easier maintenance.

Comment (FunkyMuse) [reply app/src/main/res/values-cn/strings.xml:2]: It's not good to keep it for future references, it doesn't add any value, i get your point but everywhere else is removed so...

Comment (sr093906) [reply app/src/main/res/values-cn/strings.xml]: @FunkyMuse Removed.

**Review by FunkyMuse — APPROVED** [review]

**Comment by FunkyMuse** [comment]
You can create a PR to include your flag and name into the readme for credits if you want

**Comment by FunkyMuse** [comment]
Thank you for the translation

**Comment by sr093906** [comment]
@FunkyMuse Thanks. No need to do so.

**Pull request merged**